crates/ra_assists/Cargo.toml

```diff
@@ -20,5 +20,6 @@ ra_fmt = { path = "../ra_fmt" }
 ra_prof = { path = "../ra_prof" }
 ra_db = { path = "../ra_db" }
 ra_ide_db = { path = "../ra_ide_db" }
+hir_expand = { path = "../ra_hir_expand", package = "ra_hir_expand" }
```

Comment:
You shoudn't use the `hir_*` crate directly. You have to reexport thing you need in `hir` crate and use it from hir crate :)